frontend/app/src/components/common/NoDataScreen.vue

```diff
@@ -22,17 +18,17 @@
   </component>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
+<script setup lang="ts">
```

Comment:
Did we only enforce setup-script on develop? or did the lint rule somehow missed this?

Reply:
IIRC we just done refactoring them in develop, not in bugfixes

Reply:
Ok for this we will have a conflict to resolve